app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script.rb

```diff
@@ -1,7 +1,11 @@
 class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScript <
   ManageIQ::Providers::ExternalAutomationManager::ConfigurationScript
+
   include ProviderObjectMixin
   include ManageIQ::Providers::AnsibleTower::AutomationManager::TowerApi
+  include SupportsFeatureMixin
```

Comment:
This is usually only done in core in the base model

Reply:
looks like it is not defined in master yet.